### PR TITLE
Theme Builder: Add chat colors

### PR DESCRIPTION
**Pull request by Strider2342**

### CODE REVIEW

**Review by EugeniyKiyashko — DISMISSED** [review]

**Comment by Strider2342** [comment]
[Chat: support themebuilder](https://trello.com/c/SXflpvOQ/9546-chat-support-themebuilder)

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 1]
```diff
@@ -4,27 +4,107 @@
...
+/**
+* $name 20. Border color
+* $type color
+*/
 $chat-border-color: $base-border-color !default;
-$chat-messagebox-border-color: $base-border-color !default;
+$chat-messagebox-border-color: $chat-border-color !default;
+
+/**
+* $name 30. Avatar color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 30. Avatar text color
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 2]
```diff
@@ -4,27 +4,107 @@
...
 $chat-avatar-color: $base-text-color !default;
+
+/**
+* $name 31. Avatar background color
+* $type color
+*/
 $chat-avatar-background-color: $base-active-bg !default;
+
+/**
+* $name 40. Bubble color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 40. Bubble text color
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 3]
```diff
@@ -4,27 +4,107 @@
...
+* $type color
+*/
 $chat-bubble-color-primary: $base-text-color !default;
+
+/**
+* $name 41. Bubble background color primary
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 41. Bubble background color (your messages)
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 4]
```diff
@@ -4,27 +4,107 @@
...
+* $type color
+*/
 $chat-bubble-background-color-primary: null !default;
+
+/**
+* $name 42. Bubble background color secondary
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 42. Bubble background color (others' messages)
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 5]
```diff
@@ -4,27 +4,107 @@
...
+* $type color
+*/
 $chat-bubble-background-color-secondary: $base-border-color !default;
+
+/**
+* $name 50. Information color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 50. Message caption color
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 6]
```diff
@@ -4,27 +4,107 @@
...
 
-$chat-typingindicator-color: null !default;
+/**
+* $name 81. Typing indicator background color
+* $type color
+*/
 $chat-typingindicator-bubble-bg-color: $chat-bubble-background-color-secondary !default;
+
+/**
+* $name 82. Typing indicator circle background color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 82. Typing indicator outer circles color
```

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 7]
```diff
@@ -4,27 +4,107 @@
...
+* $type color
+*/
 $chat-typingindicator-circle-bg-color: null !default;
+
+/**
+* $name 83. Typing indicator center circle background color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 83. Typing indicator central circle color
```

Comment (vladaskorohodova) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: Can we also move this one above "outer circles"?

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss [hunk 8]
```diff
@@ -60,7 +139,12 @@ $chat-typingindicator-circle-bg-color-center: null !default;
   $chat-messagelist-empty-icon-background-color: lighten($base-bg, 7.84) !default;
   $chat-messagelist-empty-prompt-color: lighten($base-label-color, 7.84) !default;
 
-  $chat-typingindicator-color: $chat-information-color !default;
   $chat-typingindicator-circle-bg-color: lighten($base-bg, 33.92) !default;
   $chat-typingindicator-circle-bg-color-center: lighten($base-bg, 43.92) !default;
 }
+
+/**
+* $name 80. Typing indicator color
```
Comment (vladaskorohodova) [inline packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]:
```suggestion
* $name 80. Typing indicator text color
```
I'm not so sure about this one, @EugeniyKiyashko can you please confirm that it's *text* color?

Comment (EugeniyKiyashko) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: Yes, it's color of the this text 
image

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 1 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 2 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 3 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 4 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 5 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 6 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 7 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

File: packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss (hunk 8 above)
Comment (Strider2342) [reply packages/devextreme-scss/scss/widgets/fluent/chat/_colors.scss]: 928565d1f0814077eb111893035b73aa0945b67b

**Review by vladaskorohodova — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**